src/Case-Channel.f90

```diff
@@ -154,6 +169,8 @@ subroutine boundary_conditions_channel (ux,uy,uz,phi)
     use param
     use var, only : di2
     use variables
+    use decomp_2d
```

Comment:
this module is loaded at the beginning of the module

Reply:
Corrected